ydb/core/kqp/ut/view/input/cases/aggregates_and_window/create_view.sql

```diff
@@ -0,0 +1,43 @@
+CREATE VIEW `/Root/aggregates_and_window` WITH (security_invoker = TRUE) AS
+    SELECT
```

Comment:
This test might be an overkill, but it uses SUM, AVG and COUNT aggregate functions as well as a WINDOW clause